lib/intern.js

```diff
@@ -0,0 +1,160 @@
+const _ = require('lodash')
+const Uuid = require('node-uuid')
+
+
+function ensureId(ent) {
+  if (undefined !== ent.id$) {
+    return ent.id$
+  }
+
+  return Uuid()
+}
+
+
+function isNew(ent) {
+  return !('id' in ent)
+}
+
+
+function escapeStr(input) {
```

Comment:
how and where is this used? to escape SQL?

Reply:
@rjrodger Legacy code is using `escapeStr` in order to prevent SQL injection. A potentially better alternative would be to use `self.connection.escape`.